protocol/wasmbinding/bindings/msg.go

```diff
@@ -1,15 +1,48 @@
 package bindings
 
 import (
-	sendingtypes "github.com/dydxprotocol/v4-chain/protocol/x/sending/types"
-
-	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+	subaccounttypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
-type SendingMsg struct {
-	CreateTransfer         *sendingtypes.MsgCreateTransfer         `json:"create_transfer,omitempty"`
-	DepositToSubaccount    *sendingtypes.MsgDepositToSubaccount    `json:"deposit_to_subaccount,omitempty"`
-	WithdrawFromSubaccount *sendingtypes.MsgWithdrawFromSubaccount `json:"withdraw_from_subaccount,omitempty"`
-	PlaceOrder             *clobtypes.MsgPlaceOrder                `json:"place_order,omitempty"`
-	CancelOrder            *clobtypes.MsgCancelOrder               `json:"cancel_order,omitempty"`
+type DydxCustomWasmMessage struct {
+	DepositToSubaccount    *DepositToSubaccount    `json:"deposit_to_subaccount,omitempty"`
+	WithdrawFromSubaccount *WithdrawFromSubaccount `json:"withdraw_from_subaccount,omitempty"`
+	PlaceOrder             *PlaceOrder             `json:"place_order,omitempty"`
+	CancelOrder            *CancelOrder            `json:"cancel_order,omitempty"`
+}
+
+type DepositToSubaccount struct {
```

Comment:
For these golang structs (which IIUC 1:1 exactly maps to Rust structs), can we version suffix (e.g. `PlaceOrderV1`), in case these schemas need to be upgraded in the future

Reply:
Will do

Reply:
Good idea. It is easy to add new cases to the DydxMsg enum later on but pretty hard to replace it entirely with a new version.

Reply:
Agreed. Will be adding versioning on rust side as well. We'll know what version of message the contract will be sending and can ensure backwards compatibility